apps/judicial-system/backend/src/app/modules/case/limitedAccessCase.controller.ts

```diff
@@ -100,7 +106,7 @@ export class LimitedAccessCaseController {
   ): Promise<Case> {
     this.logger.debug(`Getting limitedAccess case ${caseId} by id`)
 
-    if (!theCase.openedByDefender) {
+    if (user.role === UserRole.DEFENDER && !theCase.openedByDefender) {
```

Comment:
This was not directly related to the task, but we agreed that it made sense to add it to avoid unexpected behaviour 👍

Reply:
Good catch 🙌